tidb-cloud/import-csv-files.md

```diff
@@ -14,44 +14,61 @@ This document describes how to import uncompressed CSV files from Amazon Simple
 
-Before importing CSV files into TiDB Cloud, you need to create the target database and table. Alternatively, TiDB Cloud can create these objects for you as part of the import process if you provide the target database and table schema as follows:
+## Step 2. Create the target table schemas
 
-1. In the Amazon S3 or GCS directory where the CSV files are located, create a `${db_name}-schema-create.sql` file that contains the `CREATE DATABASE` DDL statement.
+Because CSV files do not contain schema information, before importing data from CSV files into TiDB Cloud, you need to create the table schemas using either of the following methods:
 
-    For example, you can create a `mydb-scehma-create.sql` file that contains the following statement:
+- Method 1: In TiDB Cloud, create the target databases and tables for your source data.
 
-    {{< copyable "sql" >}}
+- Method 2: In the Amazon S3 or GCS directory where the CSV files are located, create the target table schema files for your source data.
 
-    ```sql
-    CREATE DATABASE mydb;
-    ```
+    1. Create database schema files for your source data.
 
-2. In the Amazon S3 or GCS directory where the CSV files are located, create a `${db_name}.${table_name}-schema.sql` file that contains the `CREATE TABLE` DDL statement.
+        If your CSV files follow the naming rules in [Step 1](#step-1-prepare-the-csv-files), the database schema files are optional. Otherwise, the database schema files are mandatory.
 
-    For example, you can create a `mydb.mytable-schema.sql` file that contains the following statement:
+        Each database schema file must be in the `${db_name}-schema-create.sql` format and contain a `CREATE DATABASE` DDL statement. With this file, TiDB Cloud will create the `${db_name}` database to store your data when you import the data.
 
-    {{< copyable "sql" >}}
+        For example, if you create a `mydb-scehma-create.sql` file that contains the following statement, TiDB Cloud will create the `mydb` database when you import the data.
 
-    ```sql
-    CREATE TABLE mytable (
-    ID INT,
-    REGION VARCHAR(20),
-    COUNT INT );
-    ```
+        {{< copyable "sql" >}}
 
-    > **Note:**
-    >
-    > The `${db_name}.${table_name}-schema.sql` file should only contain a single DDL statement. If the file contains multiple DDL statements, only the first statement takes effect.
+        ```sql
+        CREATE DATABASE mydb;
+        ```
+
+    2. Create table schema files for your source data.
+
+        The table schema files are mandatory. If you do not include the files in the Amazon S3 or GCS directory where the CSV files are located, TiDB Cloud will return an error when you import the data.
```

Comment:
This should be "If you do not include the files in the Amazon S3 or GCS directory where the CSV files are located and the corresponding table is not manually created in TiDB in advance.  "

Reply:
```suggestion
        The table schema files are mandatory. If you do not include the files in the Amazon S3 or GCS directory where the CSV files are located, TiDB Cloud will not create the corresponding tables for you when you import the data.
```